source/_integrations/bluetooth.markdown

```diff
@@ -19,6 +19,10 @@ The Bluetooth integration will detect nearby Bluetooth devices. Discovered devic
 
 {% include integrations/config_flow.md %}
 
+## Before you begin
+
+The Linux Kernel can occasionally cause Bluetooth problems as it doesn't prioritize stability of the Bluetooth stack. Some users have found that a better approach than a directly connected adapter or card is to use a Bluetooth proxy by means of an ESP32. This is particularly of interest to users who virtualize their instance, where the USB pass thru may cause additional problems. More information is available in the Remote Adapters section below or by visiting ESPhome's [Bluetooth proxy page](https://esphome.github.io/bluetooth-proxies/).
```

Comment:
```suggestion
In many cases, a better approach than a directly connected adapter or card is to use a Bluetooth proxy using an ESP32 since Linux kernel updates have previously broken Bluetooth functionality and Bluetooth driver support Linux generally falls behind other operating systems for newer adapters. A Bluetooth proxy is particularly interesting to users who virtualize their instance, where the USB pass-through may cause additional problems. More information is available in the Remote Adapters section below or by visiting ESPhome's [Bluetooth proxy page](https://esphome.github.io/bluetooth-proxies/). 

Suppose a Bluetooth proxy is not a good fit for your use case. Consider using the Home Assistant Operating System when using a local adapter because it includes Bluetooth patches for issues unsolved in other operating systems.
```

Reply:
I adjusted the text a bit to make it more factual and tried to take opinions out since we are target documentation.

Reply:
LGTM 👍 I think having such information near the top of the page will help reduce support requests when the Linux kernel breaks again. Even though it’s mentioned under remote adapters, the page is quite long and that section is probably skimmed over because it’s benefits are not clear. This should hopefully rectify that.

Reply:
We do get a LOT of support requests for this type of situation. There is no way I can answer all of them so having it in the docs is a good idea

Reply:
My thinking exactly. Thanks for merging and for the input on my wording ;)